pkg/systemd/parser/unitfile.go

```diff
@@ -932,3 +932,13 @@ func (f *UnitFile) GetTemplateParts() (string, string) {
 	}
 	return parts[0] + "@" + ext, parts[1]
 }
+
+func PathEscape(path string) string {
+	var escaped strings.Builder
+	if wordNeedEscape(path, true) {
```

Comment:
calling this seems a pit pointless? In escapeString() we have to iterate anyway over all chars so just doing a single pass in escapeString() seems better? Ok we have to allocate a extra string buffer in the good case but I don't think this is a big deal.

If you prefer to keep it like this would at least restructure the if else here
```
if !wordNeedEscape {
    return path
}
escapeString()

```

to remove the else block

Reply:
Sure, removed.